.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Continuous Integration and Deployment
+on:
+  push:
+    branches:
+      - setup-github-actions-for-ci-cd
```

Comment:
```suggestion
  push
```

cf https://github.com/LesGars/dont-slack-evil/blob/master/.github/workflows/continuous_production.yml

Reply:
Yes, I did that to test on my branch. This is a draft PR, I didn't mark it as ready for review.